src/filefingerprint.cpp

```diff
@@ -107,16 +104,6 @@ FileFingerprint *FileFingerprint::unserialize(string *d)
     return fp;
 }
 
-FileFingerprint& FileFingerprint::operator=(FileFingerprint& rhs)
-{
-    isvalid = rhs.isvalid;
-    size = rhs.size;
-    mtime = rhs.mtime;
-    memcpy(crc, rhs.crc, sizeof crc);
-
-    return *this;
-}
-
```

Comment:
it's a lot easier to review, and to see what's happened in the history, if we don't move these functions around

Reply:
I think the order of functions in the header should match that in the cpp. We should clean this up whenever we change code in this area. This will make reading this module easier and this review only needs to done once.

Reply:
I moved it back where it was for now.